contracts/MedianOracle.sol

```diff
@@ -186,10 +186,10 @@ contract MedianOracle is OwnableUpgradeable, IOracle {
                     .timestamp;
                 if (reportTimestampPast < minValidTimestamp) {
                     // Past report is too old.
-                    emit ReportTimestampOutOfRange(providerAddress);
+                    // Deprecated: emit ReportTimestampOutOfRange(providerAddress);
```

Comment:
I think we can just remove it, rather than commenting it out. Dont think any of our off-chain services rely on this log.

Reply:
I think it is a good idea to leave so that previous logs are still understandable